### PR TITLE
global: fixes and tx configuration

### PR DESCRIPTION
**Pull request by alizeepace**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -61,6 +61,7 @@
 ]
 
 install_requires = [
+    'Flask>=0.11.1',
     'invenio-db[versioning]>=1.0.0a9',
```
Comment (jirikuncar) [inline setup.py]: we might need to bump inveno-db to 1.0.0b1

**Comment by jirikuncar** [comment]
- [ ] missing `invenio_i18n.messages` entrypoint in `setup.py`

**Comment by alizeepace** [comment]
@jirikuncar the message catalog is empty, shouldn't I simply remove I18N features?

**Comment by jirikuncar** [comment]
> the message catalog is empty, shouldn't I simply remove I18N features?

IMHO yes, this module should not contain any user facing messages.

--
check all `git grep -i babel`

**Pull request merged**